projects/bootloader/test/test_config.c

```diff
@@ -103,7 +103,7 @@ void setup_test(void) {
   s_test_config_2.crc32 = crc32_arr((uint8_t *)&s_test_config_2, sizeof(BootloaderConfig));
   persist_commit(&s_test_persist_storage_2);
 
-  TEST_ASSERT_OK(config_verify());
+  TEST_ASSERT_OK(config_init());
 }
 void teardown_test(void) {}
 
```

Comment:
I would like one test for the default config changes - essentially, use the `flash` module to clear the two flash pages `BOOTLOADER_CONFIG_PAGE_1_FLASH_PAGE` and `BOOTLOADER_CONFIG_PAGE_2_FLASH_PAGE`, call `config_init()`, and verify that a) the config you then get from `config_get()` is the default config and b) the default config was correctly recorded on flash. You can do this by using the `persist` module to read from flash, or you can just call `config_init()` again and make sure that the config from `config_get()` is still the default, because it will read the config pages from flash.